templates/components/entry_comments_nested.html.twig

```diff
@@ -3,19 +3,21 @@
         {{ component('entry_comment', {
             comment: reply,
             showNested: false,
+            dateAsUrl: true,
```

Comment:
if the default is `true` why overwrite it?

Reply:
removed all the instances where it would match the default